modules/3d-tiles/src/tz3-loader.ts

```diff
@@ -17,7 +17,7 @@ export type TZ3LoaderOptions = LoaderOptions & {
 /**
  * Loader for 3tz packages
  */
-export const TZ3Loader: LoaderWithParser<Buffer, never, TZ3LoaderOptions> = {
+export const TZ3Loader: LoaderWithParser<ArrayBuffer, never, TZ3LoaderOptions> = {
```

Comment:
I am very confused about the `Buffer` type. It is named similar to the Node.js type, but is actually our own type and seems to do the same thing as a typed array or ArrayBuffer. Let's see if we can avoid it, if not give it a more distinctive name,